spacy_experimental/coref/tests/test_span_resolver.py

```diff
@@ -31,12 +32,12 @@
                     (25, 33, "MENTION"),     # red ball
                     (47, 49, "MENTION"),     # it
                 ],
-                f"coref_head_clusters_1": [
+                f"{DEFAULT_CLUSTER_HEAD_PREFIX}_1": [
```

Comment:
I meant entirely overriding the default prefix, so a prefix like `custom_test_prefix instead of the one used within the component.

It might be too much of a pain for the way the test suite is constructed, but parameterizing it so that it tests both the default prefix and a custom prefix would be great.

Reply:
The `test_custom_labels` test I recently added at the bottom should do that? It uses the default label as a base by modifying the training data defined here to prefix an `"x"` to all the span keys. If it's better to use a completely different label I can do that, for example by redefining the training data with different labels.

Reply:
Ah, I misunderstood among the other changes. I feel like this is a pretty confusing way to define things. Maybe a function that returns the train data given `nlp` + a prefix?

Reply:
It seems like it would be nicer to parameterize this somehow, but I think that may be too convoluted with the config+scorer settings on top.

Reply:
Ah, I see what you mean. Parametrizing it on just the prefix - basically just regenerating the `TRAIN_DATA` - is very easy to do, and allows cleaning up the `custom_label` test. 

I could also pass in `nlp` and generate the list of Examples, but I do need to modify their contents for things like the tokenization mismatch and whitespace mismatch tests, so I think it's cleaner to not do that. 

I'll go ahead and push the changes I have, let me know how they look.

Reply:
Actually, let me modify this to use pytest fixtures...